src/components/inputs/localized-text-input/localized-text-input.styles.js

```diff
@@ -16,22 +17,36 @@ const getLocalizedInputStyles = props => [
   `,
 ];
 
-const getLanguageLabelStyles = () => css`
-  /* avoid wrapping label onto new lines */
-  flex: 1 0 auto;
-  box-sizing: border-box;
-  color: ${vars.fontColorDisabled};
-  height: ${vars.sizeHeightInput};
-  line-height: ${vars.sizeHeightInput};
-  background-color: ${vars.backgroundColorInputDisabled};
-  border-top-left-radius: ${vars.borderRadiusInput};
-  border-bottom-left-radius: ${vars.borderRadiusInput};
-  border: 1px ${vars.borderColorInputDisabled} solid;
-  padding: 0 ${vars.spacing8};
-  transition: ${vars.transitionStandard};
-  border-right: 0;
-  box-shadow: none;
-  appearance: none;
-`;
+const getLanguageLabelStyles = (props, theme) => {
```

Comment:
even those we aren't using `props` here, I added it for consistency sake with the other methods.